README.md

```diff
@@ -178,6 +178,71 @@ excludes all algorithms of the "Sphincs" family.
 *Note*: By default, interoperability testing with oqs-openssl111 is no longer
 performed by default but can be manually enabled in the script `scripts/runtests.sh`.
 
+Building under Windows
+--------------------
+The instructions below were tested on Windows 10 and 11 using MSYS2 MINGW64 (building with Visual Studio 2019 was NOT successful in our case). They are a bit experimental and only provisional. Note that the tests were skipped in the building process, however, setting up a test server and client with PQC algorithms still works.
+
+## Pre-requisites
+
+Open MSYS2 MINGW64 and make sure that git is installed:
+
+    pacman -S git
+
+Create a new directory `.local`:
+
+    mkdir install
+
+## OpenSSL 3
+
+Build and install OpenSSL 3 in `.local`:
+
+    git clone git://git.openssl.org/openssl.git
+    cd openssl
+    ./config --prefix=$(echo $(pwd)/../.local) && make && make install_sw
+    cd ..
+
+For [OpenSSL implementation limitations, e.g., regarding provider feature usage and support,
+see here](https://wiki.openssl.org/index.php/OpenSSL_3.0#STATUS_of_current_development).
+
+
+## liboqs
+
+Build and install `liboqs` in `.local`:
+
+    git clone https://github.com/open-quantum-safe/liboqs.git
+    cd liboqs
+    cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/../.local -S . -B _build
+    cmake --build _build && cmake --install _build
+    cd ..
+
+Further `liboqs` build options are [documented here](https://github.com/open-quantum-safe/liboqs/wiki/Customizing-liboqs).
+
+## Building the provider
```

Comment:
The following are actual code changes that would be good to have in the baseline code (if they don't break it): Does anything speak against adding them as-is to the files as documented?

Reply:
Thanks for the constructive feedback. Since I am not too familiar with cmake and the building process, I was reluctant to add the following to the source code. To my knowledge it should work under Linux as well so I will change the source code.